src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -668,12 +689,7 @@ private static bool IsNextExpressionPotentiallyAmbiguous(ExpressionSyntax node)
         }
 
         private static bool IsSimpleOrDottedName(ExpressionSyntax expression)
-        {
-            return expression.IsKind(
-                SyntaxKind.IdentifierName,
-                SyntaxKind.QualifiedName,
-                SyntaxKind.SimpleMemberAccessExpression);
-        }
+            => expression.Kind() is SyntaxKind.IdentifierName or SyntaxKind.QualifiedName or SyntaxKind.SimpleMemberAccessExpression;
```

Comment:
Speaking of "coding conventions we should (decide on and) document", wasn't there recently a PR that changed everything _into_ to the `IsKind(..)` form?

Reply:
:)   IN terms of the two method call forms, i think we prefer IsKind.  However, i just prefer patterns here entirely :)